torch/_dynamo/variables/functions.py

```diff
@@ -363,6 +363,29 @@ def call_function(
         args: "list[VariableTracker]",
         kwargs: "dict[str, VariableTracker]",
     ) -> "VariableTracker":
+        # Handle a `mark_traceable(fn)` call
+        if self.fn is torch._dynamo.mark_traceable:
+            raise_type_error = True
+            if len(args) == 1 and len(kwargs) == 0:
```

Comment:
is it possible to do the error handling in a function that Dynamo then inlines into? Otherwise there's an edge case where the user  passes no args but the function as a kwarg

Reply:
Ugh python...

Can't think of an easy way to do that, so I just followed how Dynamo handles such arity errors in other cases -- `inspect.signature`, and added a test case that uses kwarg.

Reply:
(I'm fine with what's in this PR)